lib/Interpreter/IncrementalJIT.cpp

```diff
@@ -212,6 +212,22 @@ IncrementalJIT::getInjectedSymbols(llvm::StringRef Name) const {
   return JITSymbol(nullptr);
 }
 
+llvm::PointerIntPair<void*, 1>
```

Comment:
It's not optional, Address can be non null, and still not cached. 
A _slightly_ more efficient std::pair\<void*,bool\>.